mantidproject/mantid
# Add default legend location to settings

This PR adds an option in settings to set a default location for legends on plots.

**Report to:** Rob Dalgliesh

**To test:**

1. Open workbench and load a workspace.
2. Plot the workspace and make sure that the legend can be dragged and moved around.
3. Open settings and go to the plot tab. 
4. change the option in legend to something else.
5. plot the workspace again and check that the legend is in the new chosen location.

Fixes #28547 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
